nopolinfo #0
location: FailureLocation{className='com.google.errorprone.bugpatterns.RestrictedApiCheckerTest', failingMethods=[com.google.errorprone.bugpatterns.RestrictedApiCheckerTest#testAllowAllDefintionsInFile], erroringMethods=[], failures=[FailureType{failureName='java.lang.AssertionError', failureDetail='Saw unexpected error on line 11. All errors:
[/Testcase.java:11: error: [RestrictedApi] test
     restrictedMethod();
                     ^
    (see foo)]', isError=false}], nbFailures=1, nbErrors=0}
status: NOPATCH
dateEnd: Tue May 09 22:20:24 CEST 2017
allocatedtime: 120minutes 
passingTime: 0minutes 
nb patches: 0
nopol context: Config{synthesisDepth=3, collectStaticMethods=true, collectStaticFields=false, collectLiterals=false, onlyOneSynthesisResult=false, sortExpressions=true, maxLineInvocationPerTest=250, timeoutMethodInvocation=2000, dataCollectionTimeoutInSecondForSynthesis=900, addWeight=0.19478, subWeight=0.04554, mulWeight=0.0102, divWeight=0.00613, andWeight=0.10597, orWeight=0.05708, eqWeight=0.22798, nEqWeight=0.0, lessEqWeight=0.0255, lessWeight=0.0947, methodCallWeight=0.1, fieldAccessWeight=0.08099, constantWeight=0.14232, variableWeight=0.05195, mode=REPAIR, type=COND_THEN_PRE, synthesis=DYNAMOTH, oracle=ANGELIC, solver=Z3, solverPath='./z3_for_linux', projectSources=[./workspace/google/error-prone/211456623/check_api/src/main/java, /root/./workspace/google/error-prone/211456623/check_api/src/main/java, ./workspace/google/error-prone/211456623/test_helpers/src/main/java, /root/./workspace/google/error-prone/211456623/test_helpers/src/main/java, ./workspace/google/error-prone/211456623/core/src/main/java, /root/./workspace/google/error-prone/211456623/core/src/main/java, ./workspace/google/error-prone/211456623/annotation/src/main/java, /root/./workspace/google/error-prone/211456623/annotation/src/main/java, ./workspace/google/error-prone/211456623/annotations/src/main/java, /root/./workspace/google/error-prone/211456623/annotations/src/main/java, ./workspace/google/error-prone/211456623/docgen/src/main/java, /root/./workspace/google/error-prone/211456623/docgen/src/main/java, ./workspace/google/error-prone/211456623/docgen_processor/src/main/java, /root/./workspace/google/error-prone/211456623/docgen_processor/src/main/java, ./workspace/google/error-prone/211456623/ant/src/main/java, /root/./workspace/google/error-prone/211456623/ant/src/main/java, ./workspace/google/error-prone/211456623/refaster/src/main/java, /root/./workspace/google/error-prone/211456623/refaster/src/main/java], projectClasspath='[Ljava.net.URL;@610db97e', projectTests=[com.google.errorprone.bugpatterns.RestrictedApiCheckerTest], complianceLevel=8, outputFolder=null, json=false}
exception: null
nbStatements: 0
nbAngelicValues: 0
ignoreStatus: NOTHING_TO_IGNORE
----------



nopolinfo #1
location: FailureLocation{className='com.google.errorprone.bugpatterns.JUnit4TestNotRunTest', failingMethods=[com.google.errorprone.bugpatterns.JUnit4TestNotRunTest#shouldNotDetectMethodsOnAbstractClass], erroringMethods=[], failures=[FailureType{failureName='java.lang.AssertionError', failureDetail='Saw unexpected error on line 5. All errors:
[/Test.java:5: error: [JUnit4TestNotRun] Test method will not be run; please add @Test annotation
  public void testDoSomething() {}
              ^
    (see http://errorprone.info/bugpattern/JUnit4TestNotRun)
  Did you mean '@Test'?]', isError=false}], nbFailures=1, nbErrors=0}
status: NOPATCH
dateEnd: Tue May 09 22:20:37 CEST 2017
allocatedtime: 120minutes 
passingTime: 0minutes 
nb patches: 0
nopol context: Config{synthesisDepth=3, collectStaticMethods=true, collectStaticFields=false, collectLiterals=false, onlyOneSynthesisResult=false, sortExpressions=true, maxLineInvocationPerTest=250, timeoutMethodInvocation=2000, dataCollectionTimeoutInSecondForSynthesis=900, addWeight=0.19478, subWeight=0.04554, mulWeight=0.0102, divWeight=0.00613, andWeight=0.10597, orWeight=0.05708, eqWeight=0.22798, nEqWeight=0.0, lessEqWeight=0.0255, lessWeight=0.0947, methodCallWeight=0.1, fieldAccessWeight=0.08099, constantWeight=0.14232, variableWeight=0.05195, mode=REPAIR, type=COND_THEN_PRE, synthesis=DYNAMOTH, oracle=ANGELIC, solver=Z3, solverPath='./z3_for_linux', projectSources=[./workspace/google/error-prone/211456623/check_api/src/main/java, /root/./workspace/google/error-prone/211456623/check_api/src/main/java, ./workspace/google/error-prone/211456623/test_helpers/src/main/java, /root/./workspace/google/error-prone/211456623/test_helpers/src/main/java, ./workspace/google/error-prone/211456623/core/src/main/java, /root/./workspace/google/error-prone/211456623/core/src/main/java, ./workspace/google/error-prone/211456623/annotation/src/main/java, /root/./workspace/google/error-prone/211456623/annotation/src/main/java, ./workspace/google/error-prone/211456623/annotations/src/main/java, /root/./workspace/google/error-prone/211456623/annotations/src/main/java, ./workspace/google/error-prone/211456623/docgen/src/main/java, /root/./workspace/google/error-prone/211456623/docgen/src/main/java, ./workspace/google/error-prone/211456623/docgen_processor/src/main/java, /root/./workspace/google/error-prone/211456623/docgen_processor/src/main/java, ./workspace/google/error-prone/211456623/ant/src/main/java, /root/./workspace/google/error-prone/211456623/ant/src/main/java, ./workspace/google/error-prone/211456623/refaster/src/main/java, /root/./workspace/google/error-prone/211456623/refaster/src/main/java], projectClasspath='[Ljava.net.URL;@4ed19540', projectTests=[com.google.errorprone.bugpatterns.JUnit4TestNotRunTest], complianceLevel=8, outputFolder=null, json=false}
exception: null
nbStatements: 0
nbAngelicValues: 0
ignoreStatus: NOTHING_TO_IGNORE
----------



